src/components/TodoItem/styles.module.sass

```diff
@@ -4,11 +4,22 @@
     padding: 25px 30px
     background-color: #eee
     border: 1px solid #888
-    margin-bottom: 25px
 
 .title
     margin: 0 0 40px
     font-family: sans-serif
+    width: 100%    
+    font-size: 30px !important
+    &__enabled
+        @extend .title
+        color: black
+    &__disabled
+        @extend .title        
+        background: transparent !important
+        color: white !important
+        border: none !important
+    @media (max-width: 575px)
```

Comment:
This breakpoint is used in various places in the app. You should define it as a variable